.github/checkgroup.yml

```diff
@@ -296,7 +296,7 @@ subprojects:
       - "src/lightning_app/*"
       - "tests/integrations_app/**"
       - "!tests/integrations_app/flagship/**"
-      - "examples/app_*/**"
+      - "examples/app/*/**"
       - "requirements/app/**"
       - "setup.py"
       - "!requirements/*/docs.txt"
```

Comment:
The app-examples workflow paths need to be updated too

Every change in the checkgroup paths should match a change in a workflows

Reply:
Done